WordPress/gutenberg
# Add screen reader instructions for navigating child blocks on block selection

## What?


This PR adds aria-description text for useBlockProps and useBlocksInnerProps.

## Why?


When screen reader users insert certain types of blocks such as a group block for example, the content is empty. There are no instructions telling users how to interact with the block they just inserted.

## How?


This PR adds aria-description with simple instructions on how to add a block or navigate child blocks.

## Testing Instructions





1. Open the post/page editor.
2. Insert a columns block and choose a variation.
3. Select the columns block.
4. Notice how the columns block now contains aria-description which guides you on how to navigate to child blocks.

## Screenshots or screencast
